docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# How to Contribute
+
+:tada: Fist off, thank you very much that you want to contribute! :tada:
+
+Please help to keep the project maintainable, easy to contribute to, and more secure by following this guide.
+
+## The Contribution Process
+
+If you want to contribute to the project, making a fork and pull request:
+
+1. Create your own fork.
+2. Clone the fork locally.
+3. Make changes in your local clone.
+4. Push the changes from local to your fork.
+5. Create a [GitHub Pull Request](https://github.com/roamingthings/spyglass/pulls) when your submission is ready to
+   to be deployed into the project.
+6. A [reviewer](contributors.md) that is ready to review your submission will assign themselves to the Pull Request on
+   GitHub. The review process aims at checking the submission for defects, completeness and overall fit into the
+   general architecture
+   of the project.
+7. After a successful review the Pull Request will be 'approved' on GitHub and will be committed to the main branch by
+   a [contributor](contributors.md).
+
+## About the Review Process
+
+Every contribution to Spyglass will be reviewed before it is merged into the main branch. The review aims to check for
+defects and to ensure that the submission follows the general style and architecture of the project.
+
+It is understood that there is not a single 'best way' to accomplish a task. Therefore, it is not intended to discuss
+if the submission is the 'best' implementation.
+
+Most of the time you will receive feedback from the review. Please be prepared to provide more information or details
+and update your submission, if required.
+
+Common aspects that a review looks for:
+
+1. Are there any defects and is the submission ready to be widely distributed?
+2. Does the submission provide real additional value to the project that will improve what users will get out of the
+   software?
+3. Does the submission include automated tests (e.g. unit test) when applicable that will ensure that the implementation
+   does what it
+   is intended to do?
+4. Is the copyright of the submission clear, compatible with the project and non-gratuitous?
+5. Commits well formatted, cover a single topic and independent?
+6. Is the documentation updated to reflect the changes?
+7. Does the implementation follow the general style of the project?
+
+## Format of Commit Messages
```

Comment:
About this...

The Header of the commit should be conformal with conventional commits and the description should be contained in the commit message body.

Remember convention commits are lowercase and in present form.